brick/brick.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+func main() {
+	var A, B, P, Q, R float64
+	fmt.Println("Enter the hole dimensions (2 numbers):")
+	fmt.Scan(&A, &B)
+	fmt.Println("Enter the brick dimensions (3 numbers):")
+	fmt.Scan(&P, &Q, &R)
+	if ((A >= R) && (B >= Q)) || ((A >= R) && (B >= P)) || ((A >= Q) && (B >= R)) || ((A >= Q) && (B >= P)) || ((A >= P) && (B >= R)) || ((A >= P) && (B >= Q)) {
```

Comment:
nit: This is absolutely working solution (you get 100% for it), though it's hard to read. I'd recommend to either group it differently:
```go
fits := a >= p && (b >= q || b >= r) ||
          a >= q && (b >= p || b >= r) ||
          a >= r && (b >= p || b >= q)
```
or maybe even try to find another solution, which would be more readable. As a hint - imagine that all inputs are sorted (a <= b, p <= q <= r). Would this allow to simplify anything?